README.md

```diff
@@ -0,0 +1,39 @@
+This is a browser-based tool used to assist in resolving duplicate urls in the PDAP Agencies Airtable.
+
+Pulls data from [PDAP Agencies Airtable](https://airtable.com/app473MWXVJVaD7Es/shr43ihbyM8DDkKx4/tblpnd3ei5SlibcCX).
+
+To utilize this tool, either utilize the PDAP Interface (not currently completed) or download a csv of the Agencies data and run initial_data_load.py to load the data into the database.
+
```

Comment:
before this step I had to `pip install -r requirements.txt`

Reply:
Added that to the README, per this comment.